test/sqs_lambda_to_batch_fargate/consumer_stack/consumer_stack.py

```diff
@@ -0,0 +1,202 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+from aws_cdk import (
+    aws_iam as iam,
+    aws_events as events,
+    aws_events_targets as targets,
+    aws_lambda as _lambda,
+    aws_lambda_event_sources as _event_sources,
+    aws_sqs as sqs,
+    aws_sns as sns,
+    aws_sns_subscriptions as subs,
+    aws_logs as logs,
+    aws_ec2 as ec2,
+    aws_ecs as ecs,
+    aws_ecr as ecr,
+    aws_batch as batch,
+    aws_batch_alpha as batch_alpha,
+    Aws,
+    Stack
+)
+from constructs import Construct
+import json
+import os
+
+# Raises KeyError if environment variable doesn't exist
+language_name = os.environ["LANGUAGE_NAME"]
+producer_account_id = os.environ["PRODUCER_ACCOUNT_ID"]
+fanout_topic_name = os.environ["FANOUT_TOPIC_NAME"] # TODO: Store value in distributed keystore
+
+class ConsumerStack(Stack):
+    def __init__(self, scope: Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        #############################################
+        ##                                         ##
+        ##                 RESOURCES               ##
+        ##                                         ##
+        #############################################
+
+        # Locate SNS topic in Weathertop Central account
+        fanout_topic_arn = f'arn:aws:sns:us-east-1:{master_account_id}:{fanout_topic_name}'
```

Comment:
Where is master_account_id coming from? And is this always us-east-1?

Reply:
fixed. region is us-east-1 for now always